bok_choy/browser.py

```diff
@@ -137,24 +137,26 @@ def save_driver_logs(driver, prefix):
     Returns:
         None
     """
-    log_types = ['browser', 'driver', 'client', 'server']
-    for log_type in log_types:
-        try:
-            log = driver.get_log(log_type)
-            file_name = os.path.join(
-                os.environ.get('SELENIUM_DRIVER_LOG_DIR'), '{}_{}.log'.format(
-                    prefix, log_type)
-            )
-            with open(file_name, 'w') as output_file:
-                for line in log:
-                    output_file.write("{}{}".format(dumps(line), '\n'))
-        except:  # pylint: disable=bare-except
-            msg = (
-                "Could not save browser log of type '{log_type}'. "
-                "It may be that the browser does not support it."
-            ).format(log_type=log_type)
-
-            LOGGER.warning(msg, exc_info=True)
+    browser_name = os.environ.get('SELENIUM_BROWSER', 'firefox')
+    if browser_name != "firefox":
+        log_types = ['browser', 'driver', 'client', 'server']
```

Comment:
Not that it matters that much, but the diff would be shorter if you instead returned from the function if `browser_name == "firefox"`; wouldn't need to indent the rest of the function.